src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -2459,5 +2444,13 @@ public virtual void ApplyStyle(ViewStyle viewStyle)
                 }
             }
         }
+
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public StoryBoard StoryBoard
```

Comment:
I think this is not related with this patch.

Reply:
To fix the CA1819, I remove the property TranstionNames,
and because the transition has been refactor, so I use a new class to restore the transtion.

The name "StoryBoard" is refer to WPF.